src/core/linq/observable/fromevent.js

```diff
@@ -77,15 +77,16 @@
    * Creates an observable sequence by adding an event listener to the matching DOMElement or each item in the NodeList.
    * @param {Object} element The DOMElement or NodeList to attach a listener.
    * @param {String} eventName The event name to attach the observable sequence.
+   * @param {Object} eventListenerOptions An object describing EventListenerOptions
    * @param {Function} [selector] A selector which takes the arguments from the event handler to produce a single item to yield on next.
    * @returns {Observable} An observable sequence of events from the specified element and the specified event.
    */
-  Observable.fromEvent = function (element, eventName, selector) {
+  Observable.fromEvent = function (element, eventName, eventListenerOptions, selector) {
```

Comment:
This is a breaking change. The `eventListenerOptions` either needs to go after the `selector` or there needs to be logic to detect the original scenario of passing the `selector` function as the 3rd argument.